odoo/tools/image.py

```diff
@@ -75,7 +75,7 @@ def image_resize_image(base64_source, size=(1024, 1024), encoding='base64', file
 
     if image.size != size:
         image = image_resize_and_sharpen(image, size)
-    if image.mode not in ["1", "L", "P", "RGB", "RGBA"]:
+    if image.mode not in ["1", "L", "P", "RGB"]:
```

Comment:
So it turns out the `image_resize_and_sharpen()` call 1 line above has the side-effect of converting the image to RGBA. Any JPEG image that gets resized with PIL 4.2+ will crash because of that.
Let's just preserve the image mode in  `image_resize_and_sharpen()`, _least surprise_ principle ;-)